README.md

```diff
@@ -424,8 +424,14 @@ Examples
    ...
    arighi@virtme-ng~> ./run.sh > result.txt
 
-   # In another terminal, copy result.txt from the guest to the host using scp:
-   arighi@gpd3~> scp -P 2222 localhost:~/result.txt .
+   # From another terminal on the host, print the ssh command used to
+   # connect to the vng guest (use this to get the SSH configuration file
+   # and the name of the guest):
+   arighi@host~> vng --ssh-client --dry-run
+   ssh -F /home/arighi/.cache/virtme-ng/.ssh/virtme-ng-ssh.conf virtme-ng,2222
+
+   # Copy result.txt from the guest to the host using scp:
+   arighi@host~> scp -F /home/arighi/.cache/virtme-ng/.ssh/virtme-ng-ssh.conf virtme-ng,2222:~/result.txt .
```

Comment:
(mmh, my previous comment was marked as outdated and no longer visible from the "files changes" tab, re-adding it so @arighi can see it as there was a question for him)

What about suggesting this command, so users can easily copy it:

    arighi@host~> scp -F ~/.cache/virtme-ng/.ssh/virtme-ng-ssh.conf virtme-ng=2222:~/result.txt .

From what I got, the path is fixed, no?

Then saying that they can get the `ssh` command by looking at the `--dry-run` command, and show the example.

Also, do you think it might be good to say that with recent systemd version, people can also use: `ssh vsock/2222`?

Reply:
Yeah initially I did the same, but then I changed it to show also that using `--dry-run` you can get the full ssh command, so we can cover two things in one command. But if you think it's more confusing than useful, we can simply add the just the scp command.

Reply:
What about writing something like this?

```
   # In another terminal, copy result.txt from the guest to the host using scp:
   arighi@host~> scp -F ~/.cache/virtme-ng/.ssh/virtme-ng-ssh.conf virtme-ng,2222:~/result.txt .

   # The SSH command can be printed using this command, and easily adapted later:
   arighi@host~> vng --ssh-client --dry-run
   ssh -F /home/arighi/.cache/virtme-ng/.ssh/virtme-ng-ssh.conf virtme-ng,2222
```

Reply:
oh that's nice, I like it! Feel free to push a commit on top or I can change my commit with that if you prefer